stack.yaml

```diff
@@ -46,15 +46,15 @@ extra-deps:
 
 # cardano-addresses-2.0.0
 - git: https://github.com/input-output-hk/cardano-addresses
-  commit: 9a64226fd749513ad0ed2e43e663f7132c92faa7
+  commit: de12008ee74faeccff396f46d985cb1066328b68
```

Comment:
Nope, let's stick to the previous commit here which corresponds to the latest release.

Reply:
Reverted.